explainer.md

```diff
@@ -54,7 +53,7 @@ and [Windows](https://docs.microsoft.com/en-us/windows/apps/design/motion/page-t
 
 # MPA vs SPA solutions
```

Comment:
The information is out of date, its not experimental anymore and you don't need a flag.